src/app/data/project-data.ts

```diff
@@ -564,6 +564,32 @@ export const projectsData = [
             "simulator"
         ]
     },
+    {
+        name: "Horizonpedia",
+        description: "App for iOS and Android with Item catalog, fake art comparison, and more!",
+        languages: ["en", "es", "de", "fr", "it", "jp", "ko", "nl", "ru", "zh", "se"],
+        logo: "",
+        logoStyle: "",
+        site: "",
+        android: "https://play.google.com/store/apps/details?id=com.wegner.horizonpedia",
+        ios: "https://testflight.apple.com/join/cYhwMhdD",
+        discord: "https://discord.gg/pQMRkMg",
+        facebook: "",
+        instagram: "https://www.instagram.com/horizonpedia/",
+        twitter: "https://twitter.com/HorizonpediaE",
+        git: "",
+        information: [
+            "art",
+            "critters",
+            "fossils",
+            "items",
+            "villagers",
```

Comment:
Remove trailing comma (this will break otherwise)

Reply:
Why will that break?

Reply:
Ah yea it won't, my mind is still set on when this was a JSON instead of a TS file.